lang/lang-es.js

```diff
@@ -0,0 +1,560 @@
+ * @type type
+ */
+const message = {
+    ActiveAll: 'Activar todo',
+    addcontext: 'Añadir marcador',
+    Addoption: 'Añadir opción',
+    addnew: 'Añadir nuevo',
+    adminusr: 'Administrador',
+    appPrerequites: 'Validar requisitos del sistema',
+    assign: 'Asignar',
+    assignDisable: 'Recuperar controles',
+    askplayMessage: '<span id="askplaymsg"> ¿Deberíamos empezar la sesión?</span><br /><span id="remaingmsg">Los datos restantes se descargarán en segundo plano.</span>',
+    askplaymsg: '<span id="askplaymsg"> "Descarga en curso, haga clic en Reproducir para comenzar </span>"',
+    audioInput: 'Entrada de audio',
+    audiolatency: 'Latencia de audio',
+    audioOutput: 'Salida de audio',
+    audioDisable: 'Dejar de silenciar',
+    audioEnable: 'Silenciar',
+    addQuestion: 'Añadir pregunta',
+    addContext: 'Añadir marcador',
+    answer: 'Respuestas',
+    action: 'Acciones',
+    audioTest: 'Su voz será grabada y reproducida para usted. \n Presione Ok y hable durante unos segundos.',
+    Back: 'Atrás',
+    vote: 'Votar',
+    bandfast: 'Conexión de red óptima',
+    bandmedium: 'Conexión de red lenta',
+    bandslow: 'Fallo en la conexión de red',
+    bandwitdhImageNotFound: 'Error al descargar la imagen, haga clic en el botón "Siguiente" para continuar.',
+    becomeTeacher: 'Convertirse en docente',
+    Binary_Limit_Exeed: 'Advertencia: Alto uso de BN',
+    bulkUserActions: 'Acciones masivas de usuario',
+    ByInstructor: 'Por instructor',
+    ByTimer: 'Por tiempo',
+    cancel: 'Cancelar',
+    canvasDrawMsg: 'Puede hacer clic en cualquier herramienta para dibujar' + 'haga clic y mantenga presionado el botón izquierdo del mouse para dibujar',
+    canvasmissing: 'Falta el Canvas en su navegador. Actualice su navegador a la última versión.',
+    chatEnable: 'Chat desactivado',
+    Chatroom: 'Sala de chat',
+    Circle: 'Circulo',
+    ClearAll: 'Limpiar todo',
+    clickheretoplay: 'Haga clic aquí para comenzar',
+    clicktoplay: 'Reproducir',
+    chatDisable: 'Activar chat',
+    chFireBrowsersIssue: 'Su navegador {virtualclass1} {virtualclass2} necesita actualizarse, Congrea admite Chrome 40+ y Firefox 35+.',
+    clearAllWarnMessageW: '¿De verdad quieres limpiar esta pizarra?',
+    clearAllWarnMessageD: '¿De verdad quieres limpiar estos dibujos?',
+    closedsDbheading: 'Cerrar el panel de documentos',
+    closeSharePresentationdbHeading: 'Cerrar el panel de presentación',
+    closevideoDashboard: 'Cerrar el panel de vídeo',
+    closeVoting: 'Votación cerrada',
+    cnmissing: 'Falta el número de fragmento',
+    CodeEditor: 'Editar código',
+    cof: 'Conexión desactivada',
+    collaborate: 'Colaborar',
+    commonChat: 'Chat común',
+    commonBrowserIssue: 'Su navegador {virtualclass1} {virtualclass2} no es compatible, utilice la última versión de Chrome y Firefox.',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 136. Maximum allowed is 120.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5529949)